.github/workflows/reviewdog-workflow.yml

```diff
@@ -136,6 +136,23 @@ jobs:
           reporter: github-pr-review
           workdir: 'nms/'
 
+  markdownlint:
+    name: markdownlint
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code.
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.ref }}
+          repository: ${{ github.event.pull_request.head.repo.full_name }}
+          fetch-depth: 0
+      - name: markdownlint
```

Comment:
Was wondering, why I got in a local run a lot of failures. From doc: `filter_mode: added` is default. Other jobs here set this explicitly. NIT^2 should this be set here too.

Reply:
- Added the line in question.
- Re-triggered CI.